js/src/PanZoom.ts

```diff
@@ -17,7 +17,10 @@ import * as widgets from '@jupyter-widgets/base';
 import * as d3 from 'd3';
 // var d3 =Object.assign({}, require("d3-selection"));
 const d3GetEvent = function () {
-  return require('d3-selection').event;
+  // In JupyterLab we can have require('d3-selection').event
+  // be a different module as the one that is used in d3.on
+  // leading to a null event, for now we can use window.event as fallback.
+  return require('d3-selection').event || window.event;
```

Comment:
Should we do the same in other files (like Graph.ts)? This d3-selection hack is done in multiple places

Reply:
Do you agree with the solution? If so, I think we could put it in utils.ts then

Reply:
Yep!